project/vision_backend_api/tests/test_deploy.py

```diff
@@ -354,7 +355,8 @@ def test_deploy_response(self):
             reverse('api:deploy_status', args=[deploy_job.pk]),
             "Response should contain status endpoint URL")
 
-    @patch('vision_backend_api.views.deploy_extract_features.run', noop_task)
+    @skip("Deprecated since deploy now a single call.")
```

Comment:
@StephenChan : Did I understand this correctly that this is deprecated due to single call? If so, should I just delete it?